### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by lilaczhou** [TangxinKevin/Keras_DRML#1]
Hello,I am trying to implement the code in Keras recently.It is very pleased to find your code.
I have several questions about loss function.In the paper,the author defined the loss function multilabel sigmoid cross-entropy like
![image](https://user-images.githubusercontent.com/45725450/59412481-1bd32280-8df0-11e9-9422-a2d0c6e7f43d.png)
but in the code you used binary cross entropy,I don't think they are the same,how about you?
